vermouth/data/force_fields/martini3001/aminoacids.ff

```diff
@@ -391,7 +635,53 @@ HIH                1
 #meta {"group": "Side chain bonds", "ifdef": "FLEXIBLE"}
 ;  i     j   funct   length  
  SC1   SC2    1       0.320  $stiff_fc
- SC1   SC3    1       0.300  $stiff_fc 
+ SC1   SC3    1       0.320  $stiff_fc 
+ SC2   SC3    1       0.270  $stiff_fc
+
+[constraints]
+#meta {"group": "Side chain bonds", "ifndef": "FLEXIBLE"}
+;  i     j   funct   length  
+ SC1   SC2    1       0.320
+ SC1   SC3    1       0.300  
```

Comment:
The bond has distance 0.32, the constraint 0.30?

Reply:
@paulocts ?

Reply:
For HSP, both should be 0.32.

Reply:
I mean, both distances SC1-SC2 and SC1-SC3... and both constraint and the bond in the flexible option. All 0.32 nm